components/restservice/config/cloudify-rest.conf

```diff
@@ -17,5 +17,6 @@
     file_server_uploaded_blueprints_folder: 'uploaded-blueprints',
     file_server_snapshots_folder: 'snapshots',
     file_server_resources_uri: '/resources',
-    insecure_endpoints_disabled: {{ ctx.node.properties.insecure_endpoints_disabled }}
+    insecure_endpoints_disabled: {{ ctx.node.properties.insecure_endpoints_disabled }},
+    maintenance_folder: '/opt/manager/maintenance'
```

Comment:
um... don't we need to add this to the rest service's create.py script so that it creates the dir?